docs/book/user-guide/production-guide/cloud-orchestration.md

```diff
@@ -0,0 +1,166 @@
+
+It is also relatively simple to create a [custom orchestrator flavor](../../stacks-and-components/custom-solutions/implement-a-custom-stack-component.md) for your use case.
+{% endtab %}
+{% endtabs %}
+
+{% hint style="info" %}
+Having trouble with setting up infrastructure? Try reading the [stack deployment](../../stacks-and-components/stack-deployment/) section of the docs to gain more insight. If that still doesn't work, join the [ZenML community](https://zenml.io/slack) and ask!
+{% endhint %}
+
+Please note that your local client needs to be authenticated to the cloud CLI, and have sufficient permissions to push a container to your container registry , and deploy a VM using Skypilot. However, it is no longer neccessary to have direct access to the remote artifact storage, as the pipeline executes in a VM on the cloud. Of course, that means the provisioned VM now needs access to the remote storage instead!
+
+It is important to understand at the infrastructure level how services authenticate with each other. This can get quite complex, and ZenML has a concept called [Service Connectors](../../stacks-and-components/auth-management/auth-management.md) that simplifies this. If you're curious, read that section of the docs!
+
+## Running a pipeline on a cloud stack
+
+Now that we have our remote artifact store registered, we can [register a new stack](understand-stacks.md#registering-a-stack) with it, just like we did in the previous chapter:
+
+{% tabs %}
+{% tab title="CLI" %}
+```shell
+zenml stack register miminal_cloud_stack -o skypilot -c cloud_container_registry -a cloud_artifact_store
+```
+{% endtab %}
+{% tab title="Dashboard" %}
+<figure><img src="../../.gitbook/assets/CreateStack.png" alt=""><figcaption><p>Register a new stack.</p></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+Now, using the [code from the previous chapter](understand-stacks.md#run-a-pipeline-on-the-new-local-stack), we run a training
+pipeline:
+
+Set the minimal cloud stack active:
+
+```shell
+zenml stack set miminal_cloud_stack
+```
+
+Run the training pipeline:
+```shell
+python run.py --training-pipeline
+```
+ 
+You will notice this time your pipeline behaves differently. After it has built the docker image with all your code, it will push that image, and run a VM on the cloud. Here is where your pipeline will execute, and the logs will be streamed back to you. So with a few commands we were able to ship our entire code to the cloud!
+
+Curious to see what other stacks you can create? The [Component Guide](../../stacks-and-components/component-guide/) has an exhaustive list of various artifact stores, container registries, and orchestrators that are integrated with ZenML. Try playing around with more stack components to see how easy it is to switch between MLOps stacks with ZenML.
```

Comment:
A bridge to the next chapter would also be nice here.

Reply:
I dont like doing bridges because it the chapters change in the future, so i intentionally didnt do it